src/progress-steps/types.js

```diff
@@ -12,6 +12,12 @@ import type {OverrideT} from '../helpers/overrides.js';
 
 export type ProgressStepsOverridesT = {
   Root?: OverrideT<*>,
+  Icon?: OverrideT<*>,
+  InnerIcon?: OverrideT<*>,
+  Tail?: OverrideT<*>,
+  Content?: OverrideT<*>,
+  Title?: OverrideT<*>,
+  Description?: OverrideT<*>,
```

Comment:
It looks like `ProgressSteps` doesn't pass down overrides to `Step` children. Also `Root` override name for `ProgressSteps` and `Step` relate to different elements.

Reply:
Sorry, missed the overrides addition above. We shoud remap the `Step`'s `Root` override  so it's not overlapped with `ProgressSteps`' `Root` override that maps to a different element.